src/firmware/README.md

```diff
@@ -1,12 +1,34 @@
 # Fimware
 
-### gps_firmware
+### Udev Rules
 
-- Designed to allow pass lat and lon coordinates from a [Adafruit Ultimate GPS Breakout v3](https://www.adafruit.com/product/746) to a PC over serial
-- Sends messages in the form `GPS,lat,lon,has_gps_fix`, where `has_gps_fix` is a 1 or 0 representing whether or not the GPS has a fix
+Udev assigns the names specified in the rule file to the devices, which helps matching dev files with devices.
```

Comment:
We should add something here explaining things here as if the person reading it has no knowledge of dev files or USB devices. My suggestion would be adding something like:
```
More simply, udev rules allow us to guarantee that no matter what order we plug USB devices in, they will always be accessible at the same path in software.
```

Reply:
changes made